go/ql/lib/semmle/go/frameworks/stdlib/Log.qll

```diff
@@ -16,16 +16,32 @@ module Log {
     }
   }
 
+  private class LogOutput extends Method {
+    LogOutput() { this.hasQualifiedName("log", "Logger", "Output") }
```

Comment:
You should also model (and test) [the top-level version](https://pkg.go.dev/log#Output).